packages/create-contentful-app/template/bin/create-app-definition.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
```

Comment:
Feels like this file needs more error handling. I see a lot of network requests that only handle happy path.